From khali@linux-fr.org Sat Nov 26 19:24:39 2005
Date: Sat, 26 Nov 2005 20:46:32 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Laurent Riffard <laurent.riffard@free.fr>
Subject: [PATCH 15/25] i2c: Drop i2c_driver.{owner,name}, 6 of 11
Message-Id: <20051126204632.23576e62.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-owner-and-name-06-oss.patch

From: Laurent Riffard <laurent.riffard@free.fr>

We should use the i2c_driver.driver's .name and .owner fields
instead of the i2c_driver's ones.

This patch updates the OSS drivers.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 sound/oss/dmasound/dac3550a.c   |    6 ++++--
 sound/oss/dmasound/tas_common.c |    6 ++++--
 2 files changed, 8 insertions(+), 4 deletions(-)

--- gregkh-2.6.orig/sound/oss/dmasound/dac3550a.c
+++ gregkh-2.6/sound/oss/dmasound/dac3550a.c
@@ -41,8 +41,10 @@ static int daca_detect_client(struct i2c
 static int daca_detach_client(struct i2c_client *client);
 
 struct i2c_driver daca_driver = {  
-	.owner			= THIS_MODULE,
-	.name			= "DAC3550A driver  V " DACA_VERSION,
+	.driver = {
+		.owner		= THIS_MODULE,
+		.name		= "DAC3550A driver  V " DACA_VERSION,
+	},
 	.id			= I2C_DRIVERID_DACA,
 	.attach_adapter		= daca_attach_adapter,
 	.detach_client		= daca_detach_client,
--- gregkh-2.6.orig/sound/oss/dmasound/tas_common.c
+++ gregkh-2.6/sound/oss/dmasound/tas_common.c
@@ -47,8 +47,10 @@ static int tas_attach_adapter(struct i2c
 static int tas_detach_client(struct i2c_client *);
 
 struct i2c_driver tas_driver = {
-	.owner		= THIS_MODULE,
-	.name		= "tas",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "tas",
+	},
 	.attach_adapter	= tas_attach_adapter,
 	.detach_client	= tas_detach_client,
 };
